src/operator/every.ts

```diff
@@ -8,15 +8,13 @@ import {Subscriber} from '../Subscriber';
 import {tryCatch} from '../util/tryCatch';
 import {errorObject} from '../util/errorObject';
 
-export function every<T>(predicate: (value: T, index: number, source: Observable<T>) => boolean,
+export function every<T>(predicate: (value: T, index: number, observable: Observable<T>) => boolean,
                          thisArg?: any): Observable<boolean> {
   const source = this;
-  let result;
-
   if (source._isScalar) {
-    result = tryCatch(predicate).call(thisArg || this, source.value, 0, source);
+    let result: boolean = tryCatch(predicate).call(thisArg || this, source.value, 0, source);
```

Comment:
Since you moved this declaration, I think you can use `const` instead of `let` here.